src/puppetlabs/puppetdb/facts.clj

```diff
@@ -69,7 +69,7 @@
 (defn unescape-string
   "Strip escaped quotes from a string."
   [^String s]
-  (if (= \" (.charAt s 0))
+  (if (= \" (first s) (last s))
```

Comment:
So this will only escape strings like `"\"foo\""` now instead of `"\"foo"` as well?